website/source/docs/commands/node/eligibility.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Add link the the commands/node.html.md.erb file

Reply:
done